app/forms/new_article_form.rb

```diff
@@ -0,0 +1,12 @@
+class NewArticlesForm
+    include ActiveModel::Model
```

Comment:
please check your default editor **indentation.**
For some reasons, this file has an indentation of 4 chars, but other files have 2.

All of them need to be 2 spaces deep.